go-controller/pkg/node/node_ip_handler_linux.go

```diff
@@ -396,7 +396,7 @@ func (c *addressManager) isValidNodeIP(addr net.IP) bool {
 		// IPs assigned to host interfaces to support the egress IP multi NIC feature must be excluded.
 		eipAddresses, err := c.getSecondaryHostEgressIPs()
 		if err != nil {
-			klog.Errorf("Failed to get secondary host assigned Egress IPs and ensure they are excluded: %v %v", err)
+			klog.Errorf("Failed to get secondary host assigned Egress IPs and ensure they are excluded: %v %v", eipAddresses, err)
```

Comment:
I guerss what actually makes sense here is to remove `eipAddresses` from the log

Reply:
ack cc @bpickard22 fyi.